leaktest.go

```diff
@@ -94,6 +94,21 @@ func interestingGoroutines(t ErrorReporter) []*goroutine {
 	return gs
 }
 
+// leakedGoroutines returns all goroutines we are considereing leaked and
+// the boolean flag indicating if no leaks detected
+func leakedGoroutines(orig map[uint64]bool, interesting []*goroutine) ([]string, bool) {
+	var leaked []string
+	leaked = make([]string, 0)
```

Comment:
you don't need to do this line!

Reply:
corrected!

Reply:
I have squashed commits with minor corrections into one introducing leakedGoroutines.